Utilities/docker

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+
+"""
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+ -------------------------------------------------------------------------
+ This script is used to bootstrap the Swift package manager build.
+
+ It does so by writing out a build task file which can be used to build a
+ stage1 package manager. That package manager is then expected to be able
+ to build itself to produce the production tools.
+
+ Note that currently this script is also responsible for building the package
+ manager in such a way that it can be installed along with the Swift package.
+ In particular, it knows how to build the runtime PackageDescription library
+ correctly and install it.  It can also build libSwiftPM, allowing clients to
+ access package manager functionality.
+"""
+
+import argparse
+import urllib2
+import os
+import subprocess
+import yaml
+
+def call(args):
+    """Prints and executes a command."""
+    print " ".join(args)
+    return subprocess.call(args)
+
+def docker_run(command_args):
+    """Runs a command in the container."""
+    call([
+        "docker",
+        "run",
+        "-it",
+        "--security-opt", "seccomp=unconfined",
+        "-v", os.getcwd() + ":/swiftpm",
+        "-w", "/swiftpm",
```

Comment:
What does this do?

Reply:
This sets `/swiftpm` as the working directory. Feels like the right thing to do so that users that use `docker run cmd` have those run inside the `swiftpm` folder.

Reply:
Ah, perfect.